.github/ISSUE_TEMPLATE/cleanup_request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: 'Cleanup request'
+about: Propose a cleanup task for this project.
+
+---
+
+/kind cleanup
+
+**What needs cleanup:**
+[A clear and concise description of what needs to be cleaned up.]
```

Comment:
WYDT about adding something to the template to include a way to specify which parts to clean up, i.e. line numbers, files, etc?

Reply:
Just added a checklist for this! Let me know what you think.